### ISSUE TITLE
relayer pk and appchain endpoint should parameterized

### ISSUE THREAD

**Issue by en** [octopus-network/octopus-relayer#2]
for example:

```
$ export RELAYER_PRIVATE_KEY=...
$ export APPCHAIN_ENDPOINT=wss://barnacle.rpc.testnet.oct.network:9944
$ npm start
```

Then we can store these sensitive variables in cloud secrets.

**Comment by en** [comment]
to facilitate local dev, the default appchain_endpoint could be `ws://localhost:9944`

**Comment by en** [comment]
implemented in 225e5b1b5d49e4d644548c65ecdd43dbb2295ca3
